argocd/applications/retail-store-cart.yaml

```
@@ -7,7 +7,7 @@ metadata:
     argocd.argoproj.io/sync-wave: "1"
 spec:
   project: retail-store
-  source:
+  source: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)